src/Microsoft.HttpRepl/Resources/Strings.Designer.cs

```diff
@@ -285,6 +294,29 @@ internal class Strings {
             }
         }
         
+        /// <summary>
+        ///   Looks up a localized string similar to Could not find script file {0}.
+        /// </summary>
+        internal static string RunCommand_CouldNotFileScriptFile {
```

Comment:
typo: RunCommand_CouldNotFileScriptFile -> RunCommand_CouldNotFindScriptFile

Reply:
Actually it's from my change. Sorry. Could you please update it for me?

Thanks!

Reply:
Changed in 82e9fa7.